src/pages/CustomerInvoicePage.js

```diff
@@ -105,16 +105,16 @@ export const CustomerInvoicePage = ({
     backingData,
   } = tableState;
 
+  const { isFinalised } = transaction;
```

Comment:
Get halfway?

Reply:
0475969e5f99e9a8c161a742cd4206e8cc030e1b kind of related to this comment